metadata_service/__init__.py

```diff
@@ -72,9 +72,7 @@ def create_app(*, config_module_class: str) -> Flask:
     api.add_resource(TableDetailAPI, '/table/<path:table_uri>')
     api.add_resource(TableDescriptionAPI,
                      '/table/<path:table_uri>/description')
-    api.add_resource(TableTagAPI,
-                     '/table/<path:table_uri>/tag',
```

Comment:
I removed this path because it currently doesn't work. The `TableTagAPI` has two method `put` and `delete`. Both of the methods require two positional arguments (`table_uri` and `tag`). In order this path to work, we would need to remove the `tag` position argument (or make it optional?).

Reply:
hey @swatro , i think the API is created for other services to push tags(Internally Lyft uses this API to push tag for ML feature table). Given the pr is for adding new unit test, I prefer to keep the API.

Sorry for documenting the path, the API is needed as metadata service not just serves FE but other services as well.

Let me know what you think.

Reply:
You're right that this PR is really about unit tests. I'll put this code back. I do think there is a problem with this path not working. But I can put up a different PR for that.

Reply:
@feng-tao I put up another [PR](https://github.com/lyft/amundsenmetadatalibrary/pull/86) to explain the problems with this API. I might be wrong, but I'm pretty sure only one path for this API is currently working